linux_install.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Some environment path variables
+CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+MOZ_DEST="$HOME/.mozilla"
+JSON_DEST="$MOZ_DEST/native-messaging-hosts"
+OLD_PATH="/home/william/scripts/ff2mpv"
+
+# Case by case copying the JSON
+if [ -d "$JSON_DEST" ]; then
```

Comment:
Nit: I'm okay with simplifying this a bit:

* Fail if `~/.mozilla` is missing completely
* Otherwise, use `mkdir -p` to avoid worrying about whether `$JSON_DEST` exists